requirements.txt

```diff
@@ -1,22 +1,22 @@
 -i https://pypi.python.org/simple
 beautifulsoup4==4.8.1
 bleach==3.1.0
-certifi==2019.9.11
+certifi==2019.11.28
 cfenv==0.5.3
 cg-django-uaa==2.0.0
 chardet==3.0.4
 dj-database-url==0.5.0
 django-webtest==1.9.7
-django==2.2.7
+django==2.2.8
```

Comment:
Seems like https://github.com/18F/tock/pull/935 is in this PR as well? I'm curious as to if these updates are necessary here.